lumigator/schemas/lumigator_schemas/experiments.py

```diff
@@ -7,19 +7,6 @@
 
 
 class ExperimentCreate(BaseModel):
-    name: str
-    description: str = ""
-    model: str
-    task: str | None = "summarization"
-    dataset: UUID
-    max_samples: int = -1  # set to all samples by default
```

Comment:
Check with Irina if the samples are set by experiment. IIRC it had to be consistent across workflows (but I may be wrong).

Reply:
Checked and confirmed that it does belong in the experiments object. Thanks for the comment, was good to double check 🚀